app/assets/stylesheets/components/_photographer_carousel.scss

```diff
@@ -0,0 +1,10 @@
+.carousel.pointer-event {
```

Comment:
What doeds the pointer event class do? And when/where is it used?

Reply:
it's used on the package show page.  I made a new component file for it but can likely take the styling for pointer-event and add it to the _carousel.scss file  @Matth93